chain/bvs.go

```diff
@@ -292,6 +293,21 @@ func (bvs *BlockValidationState) VerifyTransactionWithBlock(txn *transaction.Tra
 				})
 			}
 		}()
+	case pb.PayloadType_GENERATE_ID_2_TYPE:
+		generateIDPayload := payload.(*pb.GenerateID2)
+		amount = common.Fixed64(generateIDPayload.RegistrationFee)
```

Comment:
Should reject transaction if generateIDPayload.RegistrationFee < config.RegFeeMinLimitation NKN